openhands/mcp/utils.py

```diff
@@ -5,6 +5,8 @@
     from openhands.controller.agent import Agent
 
 
+from mcp import McpError
```

Comment:
```suggestion
from fastmcp.exceptions import FastMCPError 
```

Could we try this from FastMCP instead?